src/features/Planner/data/conversion_helpers.ts

```diff
@@ -0,0 +1,31 @@
+import throwAnyGraphQLErrors from "util/throwAnyGraphQLErrors";
+import { ensureInt } from "global/utils";
+
+export const handleErrors = (error) => {
+    throwAnyGraphQLErrors(error);
```

Comment:
@barneyb is this what you meant?

Reply:
Yeah! Though now that you reminded me there's top-level error handling, I'm pretty sure I biffed it.